index.html

```diff
@@ -0,0 +1,239 @@
+                                <ul id="black-right-list" class="none">
+                                    <li class="black-top-right inline">
+                                        <a class="black-link no-underline" href="#">About</a>
+                                    </li>
+                                    <li class="black-top-right inline">
+                                        <a class="black-link no-underline" href="#">Advertise</a>
+                                    </li>
+                                    <li class="black-top-right inline">
+                                        <a class="black-link no-underline" href="#">Jobs</a>
+                                    </li>
+                                    <li class="black-top-right inline">
+                                        <a class="black-link no-underline" href="#">Contact</a>
+                                    </li>
+                                </ul>
+                            </div>
+                        </div>
+                        <div class="header-line" id="white-header-line">
+                            <div class="left-nav-div" id="white-left-nav-div">
+                                <ul class="none" id="white-left-list">
+                                    <li class="white-top-left inline">
+                                        <a class="white-link no-underline" href="#">LATEST</a>
+                                    </li>
+                                    <li class="white-top-left inline">
+                                        <a class="white-link no-underline" href="#">HARD FORK</a>
+                                    </li>
+                                    <li class="white-top-left inline">
+                                        <a class="white-link no-underline" href="#">PLUGGED</a>
+                                    </li>
+                                    <li class="white-top-left inline">
+                                        <a class="white-link no-underline" href="#">README</a>
+                                    </li>
+                                    <li class="white-top-left inline">
+                                        <a class="white-link no-underline" href="#">GROWTH QUARTERS</a>
+                                    </li>
+                                    <li class="white-top-left inline">
+                                        <a class="white-link no-underline" href="#">SHIFT</a>
+                                    </li>
+                                    <li class="white-top-left inline">
+                                        <a class="white-link no-underline" href="#">NEURAL</a>
+                                    </li>
+                                </ul>
+                            </div>
+                            <div class="right-nav-div" id="white-right-nav-div">
+                                <img src="img/magnifying_glass30x30.png" alt="magnifier-icon">
```

Comment:
## Code implementation details
- [ ] Ensure the search button(icon) is also a link.